From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: VerteDinde <keeleymhammond@gmail.com>
Date: Tue, 9 Aug 2022 14:05:55 -0700
Subject: fix: cast pwrite64 arg to long for arm

This patch cases pwrite64 arg to a long in order to avoid a
compilation error on arm. This patch can be eliminated when the
upstream patch is merged into chromium main:

https://chromium-review.googlesource.com/c/linux-syscall-support/+/3786946

diff --git a/linux_syscall_support.h b/linux_syscall_support.h
index e4e816f4bd001a6de7b98972d1cbaec9aaa8f821..5ea1295f8e9bb5312469dbc1aadb67dd620d35d3 100644
--- a/linux_syscall_support.h
+++ b/linux_syscall_support.h
@@ -4842,7 +4842,7 @@ struct kernel_statfs {
                            unsigned, o2)
       LSS_INLINE _syscall5(ssize_t, _pwrite64,       int,         f,
                            const void *,   b, size_t, c, unsigned, o1,
-                           long, o2)
+                           unsigned, o2)
       LSS_INLINE _syscall4(int, _readahead,          int,         f,
                            unsigned,       o1, unsigned, o2, size_t, c)
     #endif
